src/app/mosh_client.html

```diff
@@ -63,6 +63,7 @@
     <td align="right" valign="top">
       <a href="https://github.com/rpwoodbu/mosh-chrome/wiki/Documentation" target="_blank">Help</a><br>
       <a href="" id="ssh-key">Add ssh key</a><br>
+      <a href="" id="prefs">Terminal preferences</a><br>
```

Comment:
How about just "Preferences"? We could put other things in there in the future that aren't hterm-related. I don't want to make tracking upstream changes too difficult, though, but I think we can manage.